azkaban-solo-server/src/main/resources/conf/azkaban.properties

```diff
@@ -25,6 +25,13 @@ executor.port=12321
 # mail settings
 mail.sender=
 mail.host=
+# following parameters are used when there is reverse proxy between azkaban server and user, set this parameters to the valies that enduser sees
```

Comment:
Thanks for adding comments here as well. 
To reduce duplication, maybe we should add a comment in the beginning of this file to refer to the Constant class for documentation?

Reply:
agree its duplicated and maybe too much comment, i can remove last lines, regarding location it makes sense keeping comment or some part of it in properties file, then its will be readable after build, to the user that seting up the configuration?